SVision-pro non-functional

Running svision-pro via the singularity container results in an error only visible during analysis and isn't caught by bioconda test script.

```
2025-06-16 17:02:47,948 [INFO]    Collecting X chrX_150000000_156030895, 6 candidate events found. Time cost 1s
2025-06-16 17:02:48,181 [INFO]    Collecting X chrX_156040895_160000000, 0 candidate events found. Time cost 0s
2025-06-16 17:02:48,226 [INFO]    Collecting Time Cost 1127s
2025-06-16 17:02:48,226 [INFO]    ****************** Step2 lite-Unet predicting **************************
Traceback (most recent call last):
  File "/usr/local/bin/SVision-pro", line 279, in <module>
    process_pool = NoDaemonPool(max(1, int(options.process_num / options.unet_cpu_num)))
  File "/usr/local/lib/python3.9/multiprocessing/pool.py", line 212, in __init__
    self._repopulate_pool()
  File "/usr/local/lib/python3.9/multiprocessing/pool.py", line 303, in _repopulate_pool
    return self._repopulate_pool_static(self._ctx, self.Process,
  File "/usr/local/lib/python3.9/multiprocessing/pool.py", line 319, in _repopulate_pool_static
    w = Process(ctx, target=worker,
  File "/usr/local/lib/python3.9/multiprocessing/process.py", line 82, in __init__
    assert group is None, 'group argument must be None for now'
AssertionError: group argument must be None for now
```

this is known upstream https://github.com/songbowang125/SVision-pro/issues/7 as an installation issue. I'm seeing it with:

- `quay.io/biocontainers/svision-pro:2.0--pyhdfd78af_0`
- `quay.io/biocontainers/svision-pro:2.3--pyhdfd78af_0`

@tdido since you wrote this recipe, have you seen the same?

--- a/recipes/svision-pro/meta.yaml
+++ b/recipes/svision-pro/meta.yaml
@@ -10,25 +10,25 @@ source:
   sha256: e22b31342c1792067880304fa6d8835aef802ce6fd775483fd869a08e211302c
 
 build:
-  number: 0
+  number: 1
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation --no-cache-dir
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir --use-pep517 -vvv
   run_exports:
     - {{ pin_subpackage('svision-pro', max_pin="x") }}
 
 requirements:
   host:
-    - python >=3.7.9
+    - python <3.8
     - pip
-    - setuptools
   run:
-    - python >=3.7.9
+    - python <3.8
     - pytorch ==1.10.1
     - numpy ==1.21.6
-    - pysam >=0.20.0
-    - scipy  ==1.7.3
-    - pillow ==9.2.0
-    - py-opencv >=4.5.3
+    - pysam
+    - scipy
+    - pillow
+    - py-opencv
 
 test:
   commands:
@@ -41,7 +41,7 @@ about:
   license_file: LICENSE
   dev_url: "https://github.com/songbowang125/SVision-pro"
   doc_url: "https://github.com/songbowang125/SVision-pro/blob/v{{ version }}/README.md"
-  summary: 'Neural-network-based long-read SV caller.'
+  summary: "Neural-network-based long-read SV caller."
   description: |
     A neural-network-based instance segmentation framework that represents genome-to-genome-level sequencing differences visually and discovers SV comparatively between genomes without any prerequisite for inference models.
 

